<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mdiWidget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mainMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionFasta"/>
     <addaction name="actionClustal"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionSequence"/>
     <addaction name="actionAlignment_2"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="menuImport"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSorry_no_help_available"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_Tabs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_all"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAlign"/>
    <addaction name="actionNewFolder"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuActions"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="mainStatus"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAlign"/>
   <addaction name="actionNewFolder"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionRulers"/>
   <addaction name="actionColors"/>
   <addaction name="actionSave_Image"/>
  </widget>
  <action name="actionAlign">
   <property name="text">
    <string>Align</string>
   </property>
   <property name="toolTip">
    <string>Align the selected sequences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Workspace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Workspace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFasta">
   <property name="text">
    <string>Fasta Alignment</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New Workspace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add Sequence</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionClustal">
   <property name="text">
    <string>Clustal Alignment</string>
   </property>
  </action>
  <action name="actionSequence">
   <property name="text">
    <string>Sequence</string>
   </property>
  </action>
  <action name="actionAlignment_2">
   <property name="text">
    <string>Alignment</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNewFolder">
   <property name="text">
    <string>Add Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Remove from Tree</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionToggle_Tabs">
   <property name="text">
    <string>Toggle Tab Mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>Close all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionSorry_no_help_available">
   <property name="text">
    <string>Sorry, no help available!</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRulers">
   <property name="text">
    <string>Rulers</string>
   </property>
   <property name="toolTip">
    <string>Show or hide sequence ruler</string>
   </property>
  </action>
  <action name="actionColors">
   <property name="text">
    <string>Colors</string>
   </property>
   <property name="toolTip">
    <string>Show or hide sequence colors</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="toolTip">
    <string>Capture current window as an image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
